### ISSUE TITLE
Get Geolocation Data - Amend running code. 

### ISSUE THREAD

**Issue by Jeff-Shapiro** [FreeCodeCamp/FreeCodeCamp#11741]
To better display use of Geo-location data. recommend the following amendment. 
[Get Geolocation Data](https://www.freecodecamp.com/challenges/get-geolocation-data)
current: - 
```
</script>
<div id = "data">
  <h4>You are here:</h4>
  
</div>
```
![image](https://cloud.githubusercontent.com/assets/20011187/20475525/72d6f12a-b020-11e6-8c9b-481744ab4692.png)

Change to: -
```
</script>
    <h4>You are here:</h4>
<div id = "data">
   
</div>
```
![image](https://cloud.githubusercontent.com/assets/20011187/20475545/8cbe47aa-b020-11e6-86e4-6bf6d9a73f57.png)

**Comment by abhisekp** [comment]
@Jeff-Shapiro Which challenge does this belong to?

**Comment by raisedadead** [comment]
Well yes. this is more of a bug, lol.. nice catch @Jeff-Shapiro.

Contributors, need to update this line of code:
https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json#L500-L501

and modify:

```
        "<div id = \"data\">",
        "  <h4>You are here:</h4>",
```

to below:

```
        "<h4>You are here:</h4>",
        "<div id = \"data\">",
```

**Comment by ubershibs** [comment]
I'll take a stab at this - I'm a  first-timer.

**Comment by mridulnagpal** [comment]
#11747
